test/cmd/admin.sh

```diff
@@ -250,3 +256,49 @@ os::cmd::expect_success_and_not_text 'oc get scc/restricted -o yaml' 'system:aut
 os::cmd::expect_success 'oadm policy reconcile-sccs --confirm'
 os::cmd::expect_success_and_text 'oc get scc/restricted -o yaml' 'system:authenticated'
 echo "reconcile-scc: ok"
+
+
+# Create test users/identities and groups
+os::cmd::expect_success 'oc login -u cascaded-user -p pw'
+os::cmd::expect_success 'oc login -u orphaned-user -p pw'
+os::cmd::expect_success 'oc login -u system:admin'
+os::cmd::expect_success 'oadm groups new cascaded-group cascaded-user orphaned-user'
+os::cmd::expect_success 'oadm groups new orphaned-group cascaded-user orphaned-user'
+# Add roles, sccs to users/groups
+os::cmd::expect_success 'oadm policy add-scc-to-user           restricted    cascaded-user  orphaned-user'
+os::cmd::expect_success 'oadm policy add-scc-to-group          restricted    cascaded-group orphaned-group'
+os::cmd::expect_success 'oadm policy add-role-to-user          cluster-admin cascaded-user  orphaned-user  -n default'
+os::cmd::expect_success 'oadm policy add-role-to-group         cluster-admin cascaded-group orphaned-group -n default'
+os::cmd::expect_success 'oadm policy add-cluster-role-to-user  cluster-admin cascaded-user  orphaned-user'
+os::cmd::expect_success 'oadm policy add-cluster-role-to-group cluster-admin cascaded-group orphaned-group'
+
+# Delete users
+os::cmd::expect_success 'oc delete user  cascaded-user'
+os::cmd::expect_success 'oc delete user  orphaned-user  --cascade=false'
+# Verify all identities remain
+os::cmd::expect_success 'oc get identities/anypassword:cascaded-user'
+os::cmd::expect_success 'oc get identities/anypassword:orphaned-user'
+# Verify orphaned user references are left
+os::cmd::expect_success_and_text     'oc get clusterrolebindings/cluster-admins -o json'    'orphaned-user'
```

Comment:
Select out the field of interest with a template.  Apply annotations have borked this in the past and there's no real reason to avoid it.